solidity/random-beacon/test/RandomBeacon.Authorization.test.ts

```diff
@@ -710,26 +716,219 @@ describe("RandomBeacon - Authorization", () => {
               randomBeacon.address,
               deauthorizingFirst
             )
-
-          await staking
-            .connect(authorizer)
-            ["requestAuthorizationDecrease(address,address,uint96)"](
-              stakingProvider.address,
-              randomBeacon.address,
-              deauthorizingSecond
-            )
         })
 
         after(async () => {
           await restoreSnapshot()
         })
 
-        it("should overwrite the previous request", async () => {
-          expect(
-            await randomBeacon.pendingAuthorizationDecrease(
-              stakingProvider.address
+        context("when change period is equal delay", () => {
+          before(async () => {
+            // this should be the default situation from the fixture setup so we
+            // just confirm it here
+            const {
+              authorizationDecreaseDelay,
+              authorizationDecreaseChangePeriod,
+            } = await randomBeacon.authorizationParameters()
+            expect(authorizationDecreaseDelay).to.equal(
+              authorizationDecreaseChangePeriod
             )
-          ).to.be.equal(deauthorizingSecond)
+          })
+
+          context("when delay passed", () => {
```

Comment:
Can we add one more test under `"when change period is equal delay"` to cover a case where the `requestAuthorizationDecrease` is called without any `increaseTime`?